update/genesis/test.go

```diff
@@ -0,0 +1,8 @@
+package genesis
```

Comment:
You can delete this file. You can use directly exportNodesSetupJson in TestStateExport_ExportNodesSetupJsonShouldExportKeysInAlphabeticalOrder

Reply:
done